requirements.txt

```diff
@@ -21,6 +21,7 @@ elasticsearch-dsl==5.3.0
 fasteners==0.14.1
 Flask==0.12.2
 future==0.16.0
+frozendict==1.2
```

Comment:
Consider vendoring or bundling this code (https://github.com/slezica/python-frozendict/blob/master/frozendict/__init__.py) instead of declaring a dependency. I think dependencies that haven't been updated in a while and don't have a lot of stars/other evidence of popularity are a potential hazard.

Reply:
Will do.

Reply:
Vendored `frozendict`